src/Controller/Component/AuthenticationComponent.php

```diff
@@ -172,9 +174,7 @@ protected function doIdentityCheck(): void
 
         $identity = $request->getAttribute($this->getConfig('identityAttribute'));
         if (!$identity) {
-            throw new UnauthenticatedException(
-                'No identity found. You can skip this check by configuring  `requireIdentity` to be `false`.'
-            );
+            throw new UnauthenticatedException($this->getConfig('unauthenticatedMessage', ''));
```

Comment:
Should we have a default message? Something like 'Authentication is required'?

Reply:
The constructor already set's one: https://github.com/cakephp/authentication/blob/048648d2e1daf73eed789404663aefee6747359e/src/Authenticator/UnauthenticatedException.php#L40-L42

Reply:
Good job past us.